### DISCUSSION TITLE
When connect to a domain, can we put the panes in the same window?

### DISCUSSION THREAD

**Discussion by joexue** [wezterm/wezterm#2847] [Ideas]
We don't need two windows for two domains, just merge all the panes into the existing window then we can easily switch between them.

**Comment by oderwat** [comment]
I actually wondered that `wezterm ssh ...` opens a new window. I am on 4-6 different servers inside one window in my iTerm2 all the time. I can do that in wezterm too, but somehow it makes no sense when I could have the multiplexer. Currently I just mosh/tmux/iTerm2 pane my way around.
